.github/workflows/npmPublish.yml

```diff
@@ -40,12 +33,79 @@ jobs:
           node-version: '16.x'
           registry-url: 'https://registry.npmjs.org'
 
+      # Don't commit the version bump, we will commit later
       - name: Update npm version
-        run: npm version patch
+        run: npm version patch --git-tag-version false
 
       - name: Set new version in GitHub ENV
         run: echo "NEW_VERSION=$(jq '.version' package.json)" >> $GITHUB_ENV
 
+      # Copied and edited from: https://github.com/joshjohanning-org/commit-sign-app/blob/main/.github/workflows/commit-sign.yml
+      - name: Created signed commit
+        run: |
+          #!/bin/bash
```

Comment:
For testing I'll allow this, but let's pipe this into a file with params if it works